### PR TITLE
Update Maintainer List

### PR DESCRIPTION
**Pull request by michaelavs**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by michaelavs** [comment]
@conda-forge-admin, please rerender

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ version: 2
 jobs:
   build:
     working_directory: ~/test
-    machine: true
+    machine:
+      image: ubuntu-2004:current
```
Comment (erogluorhan) [inline .circleci/config.yml:11]: I am just wondering why this change was needed

File: .scripts/build_steps.sh [hunk 2]
```diff
@@ -24,7 +24,10 @@ export CONFIG_FILE="${CI_SUPPORT}/${CONFIG}.yaml"
 cat >~/.condarc <<CONDARC
 
 conda-build:
- root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
+  root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
+pkgs_dirs:
+  - ${FEEDSTOCK_ROOT}/build_artifacts/pkg_cache
+  - /opt/conda/pkgs
```
Comment (erogluorhan) [inline .scripts/build_steps.sh:30]: The same here: could you please clarify these changes?

File: .scripts/build_steps.sh [hunk 3]
```diff
@@ -46,6 +49,10 @@ make_build_number "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
 ( endgroup "Configuring conda" ) 2> /dev/null
 
+if [[ -f "${FEEDSTOCK_ROOT}/LICENSE.txt" ]]; then
+  cp "${FEEDSTOCK_ROOT}/LICENSE.txt" "${RECIPE_ROOT}/recipe-scripts-license.txt"
+fi
+
```
Comment (erogluorhan) [inline .scripts/build_steps.sh:55]: This one as well

File: build-locally.py [hunk 4]
```diff
@@ -86,12 +86,19 @@ def main(args=None):
...
+        ):
+            run_docker_build(ns)
+        elif ns.config.startswith("osx"):
+            run_osx_build(ns)
+    finally:
+        recipe_license_file = os.path.join(
+            "recipe", "recipe-scripts-license.txt"
+        )
+        if os.path.exists(recipe_license_file):
+            os.remove(recipe_license_file)
```
Comment (erogluorhan) [inline build-locally.py:101]: The same as above

**Review by erogluorhan — COMMENTED** [review]
I am sure the PR is good to go but have a few inline questions about some changes.

File: .circleci/config.yml (hunk 1 above)
Comment (michaelavs) [reply .circleci/config.yml:11]: These changes come from re-rendering with the latest conda-smithy. From my understanding, it is just good practice that when you open a PR in a feedstock, you should re-render to make sure all of the common feedstock files are up to date with any new changes conda-forge has had since the last PR/render. Overall, the only files I directly changed (across all of the feedstocks, not just this one) are the meta.yaml, README.md, and CODEOWNERS, everything else is from the re-render called in the comment above

**Review by erogluorhan — APPROVED** [review]
